sycl/source/detail/kernel_impl.hpp

```diff
@@ -214,6 +214,18 @@ inline context kernel_impl::get_info<info::kernel::context>() const {
 template <typename Param>
 inline typename Param::return_type
 kernel_impl::get_info(const device &Device) const {
+  if constexpr (std::is_same_v<
+                    Param, info::kernel_device_specific::global_work_size>) {
+    bool isDeviceCustom = Device.get_info<info::device::device_type>() ==
+                          info::device_type::custom;
+    if (!isDeviceCustom && !isBuiltInKernel(Device))
+      throw exception(
+          sycl::make_error_code(errc::invalid),
+          "\"info::kernel_device_specific::global_work_size\" descriptor may "
```

Comment:
I'm not sure how common it is for us to use `\"` in our exception messages. If there are other examples of it, I will concede, but seeing as `device_type::custom` is mentioned below without it, I would prefer we just exclude them.

Reply:
I'm ok to remove it [ec5c789](https://github.com/intel/llvm/pull/9072/commits/ec5c7898897c9d3592556d85e8803d3c769e7631)